pkg/cmd/roachtest/tests/db_console.go

```diff
@@ -158,8 +159,10 @@ func (d *dbConsoleCypressTest) buildDockerImage(ctx context.Context) {
 	require.NoError(t, rtCluster.Install(ctx, t.L(), workloadNode, "docker"), "failed to install docker")
 
 	// Build docker image on the workload node.
-	require.NoError(t, rtCluster.RunE(ctx, option.WithNodes(workloadNode),
-		fmt.Sprintf("docker build -t %s %s", d.imageName, d.cypressWorkingDir)))
+	require.NoError(t, testutils.SucceedsSoonError(func() error {
```

Comment:
Can we just use `testutils.SucceedsSoon`?

Reply:
Done